src/main/kotlin/nl/info/zac/notification/Channel.kt

```diff
@@ -21,7 +21,8 @@ enum class Channel(private val code: String, val resourceType: Resource) {
     INFORMATIEOBJECTTYPEN("informatieobjecttypen", Resource.INFORMATIEOBJECTTYPE),
     OBJECTEN("objecten", Resource.OBJECT),
     ZAKEN("zaken", Resource.ZAAK),
-    ZAAKTYPEN("zaaktypen", Resource.ZAAKTYPE);
+    ZAAKTYPEN("zaaktypen", Resource.ZAAKTYPE),
+    TEST("test", Resource.TEST);
```

Comment:
These notification channels (and also resources, actions etc) are predefined in the 'Notificatie API' specification. With this we are deviating from the official specification. I don't think that is a good idea. What is the purpose of this TEST channel and resource?

PS: these Notificatie specs are currently very hard to find and for some reason not really defined as part of the ZGW API but separately. Still these are specs that we (and OpenZaak of course) use. Ideally we should even generate client stub code from these specs like we do for the other APIs we integrate with at some point. See the `Notificatie API specificaties` links on: https://dimpact.atlassian.net/wiki/spaces/PZW/pages/4523949/Links

Reply:
@infojohn was trying to test the callback URL status with OpenNotificaties:
![image](https://github.com/user-attachments/assets/e590db87-84b8-4755-a350-4b0407021d75)

Seems Test channel is part of both our TEST and local environment and used by the UI to check is URL is reachable

Reply:
I don't see a 'test' channel mentioned in this screenshot? Or do you mean that this `Check the status of the callback URLs..` button implicitly uses this `test` channel?

Reply:
>Seems Test channel is part of both our TEST and local environment and used by the UI to check is URL is reachable

Shouldn't we explicitly check this with Maykin instead of assuming that this is how it should be done?

Seems maybe more like a story to me coming from the integration team instead of a task?

Reply:
In any case, if we really need to do this then I would add comments everywhere explaining what the purpose of this `TEST` channel, resource etc is, because other developers in future will surely have the same question as I have (I hope..).